fossevents/templates/email/event_creator_email.html

```diff
@@ -0,0 +1,14 @@
+{% load staticfiles i18n %}
+
+Hi,
+
+Thank you for visiting our website.
```

Comment:
Updated copy to use:

Subject: `FOSSEvents | "{{ event.title|truncatechars:15 }}" published!`
Body: 
```
Hi {{ first_name }},

Your event titled "{{ event.name }}" is published and public now. You can share the event using the following url: 

{{ event.get_absolute_url }}

Here is the link you can use to edit the event: {{ event.get_edit_url }}. You can share this link with other whom you want to provide edit access.

If you are having any issues or got any feedback, you can reply to this email and someone from the team will reach out to you. 

Cheers,
FOSSEvents.in
(FOSSEvents is a open-source project, run by voluteers)
```